admin/horace_install.m

```diff
@@ -0,0 +1,134 @@
+function horace_install()
+%HORACE_INSTALL install this instance of Horace
+%
+HORACE_ON_SUB_PATTERN = '${Horace_CORE}';
+HERBERT_ON_SUB_PATTERN = '${Herbert_CORE}';
+
+install_root = fileparts(mfilename('fullpath'));
+
+% Find directories containing herbert/horace_init.m
```

Comment:
This function is doing the same thing twice:
- find xxx_on
- reat content
- replace placeholder with xxx_init_dir
- save file to userpath

Rather than duplicating the code and interlacing it, two function calls to a single implementation would be easier to follow